interface/src/app/mlmodel/validator.tpl.html

```diff
@@ -0,0 +1,5 @@
+<div class="alert alert-danger" role="alert" ng-if="$ctrl.errMessage">
+  {{$ctrl.errMessage}}
+  <br>
+  Please click <a href="#/home">Home</a> to reset it.
```

Comment:
This works fine as an initial implementation -- you had a ton of work as it was -- and the overall workflow is much better. I don't want to lose sight of the fact that the flow could be even better.

Note that I prioritized the fix as "before multi-organism." Maybe it should say "before multi-MLModel." That's when I think it will really start to matter.